src/block_operations.jl

```diff
@@ -19,8 +18,7 @@
     end
     @sync begin
         Threads.@spawn begin
-            #_mul!(multithreaded,     C11, A11, B11, sz)
-            gemm_kernel!(C11, A11, B11)
```

Comment:
`gemm_kernel!` doesn't look too appropriate here anyway, as it doesn't recurse meaning we don't get cache reuse.